azure-communication-ui/chat/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="title_activity_chat">ChatActivity</string>
```

Comment:
All resource should start with library name prefix: azure_communication_ui_chat_